### PR TITLE
commit2

### PR DESCRIPTION
**Pull request by Nechami-Malin**

### CODE REVIEW

File: new.html [hunk 1]
```diff
@@ -8,5 +8,6 @@
 <body>
     <h1>hello</h1>
     <h1>nechami malin!!!!</h1>
+    <H2>Yaffi Wertaimer visited Nechami Malin😜</H2>
```
Comment (Nechami-Malin) [inline new.html:11]: code review

**Review by Nechami-Malin — COMMENTED** [review]
hello

**Comment by Nechami-Malin** [comment]
2

**Review by Nechami-Malin — COMMENTED** [review]
try!!!!!!!!

**Review by yaffiWertaimer — APPROVED** [review]
try!!!!!!!!!

**Pull request merged**